README.md

```diff
@@ -957,6 +957,10 @@ Return time from now.
 moment(1536484369695).fromNow();
 // => "4 days ago"
 
+// Native
+new Intl.RelativeTimeFormat().format(4, 'day');
```

Comment:
`4` should be `-4`
🤦 
Fixed in https://github.com/you-dont-need/You-Dont-Need-Momentjs/pull/94